distribution/src/conf/synapse.properties

```diff
@@ -66,3 +66,7 @@ mediation.flow.statistics.collect.all=false
 
 # Script Mediator Pool (impatcts only external scripts)
 #synapse.script.mediator.pool.size=15
+
+# Internal APIs
+internal.http.api.enabled=true

```

Comment:
Is there a way to hide this property to improve the user experience. For a user, this property does not make sense.